src/helper-methods.js

```diff
@@ -0,0 +1,67 @@
+import merge from "lodash/object/merge";
+import { Chart, Style } from "victory-util";
+import { Transform } from "react-art";
+
+
+module.exports = {
+  getStyles(props, defaultStyles) {
+    const style = props.style ? merge({}, defaultStyles, props.style) : defaultStyles;
+    return Style.removeInvisible(Chart.evaluateStyle(style));
+  },
+
+  getContentLength(props) {
+    if (props.children) {
```

Comment:
IMO short circuiting is preferred since it prevents methods that grow from being one big if statement.

```
if (!props.children) { return 1; }

// do stuff
```